teos/cli/teos_cli.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
```

Comment:
I'm not gonna get to picky since the cli needs a followup anyway, but I think the responses should be at least properly formatted for now. Currently this is returning the whole JSON-RPC response, whereas it should only return the "result" field. e.g:

```
teos/cli/teos_cli.py get_all_appointments

{
    "id": 96225753724115027851564137027831193896,
    "jsonrpc": "2.0",
    "result": {
        "responder_trackers": {},
        "watcher_appointments": {}
    }
}

Reply:
Had already done this in the other branch indeed; ported here as well with cee0d72.